vendor.conf

```diff
@@ -42,6 +42,8 @@ github.com/syndtr/gocapability d98352740cb2c55f81556b63d4a1ec64c5a319c2
 gotest.tools v2.3.0
 github.com/google/go-cmp v0.2.0
 go.etcd.io/bbolt v1.3.2
+go.opencensus.io v0.21.0
+github.com/hashicorp/golang-lru v0.5.1
```

Comment:
This must be indirect?

Reply:
Yes, it's used by OC I think. Should I treat it different in this case?

Reply:
Nah just checking